shared/haml/course_explorer_table.haml

```diff
@@ -181,12 +186,12 @@
                   %br/
                   =course[:description2]
             - if course[:link]
-              %a{href: course[:link], style: "text-decoration: none"}
+              %a.linktag{id: "link-#{course[:id]}", href: course[:link], style: "text-decoration: none"}
                 %button.launch{style: "font-size: 12px"}
                   Learn more
             - if course[:course_link]
               &nbsp;
-              %a{href: course[:course_link]}
+              %a.linktag{id: "course-link-#{course[:id]}", href: course[:course_link]}
                 %button.course-explorer-gray-button{style: "font-size: 12px"}
                   View course
         .arrow_box_close{style: "position: absolute; right: 10px; top: 0px; cursor: pointer", data: {courseindex: course[:regular_order]}}
```

Comment:
That's cool we can track open/close too! Do we also count clicking 'x' as a close? I don't think we need to distinguish it from clicking the block, but would like to make sure that a close counts either method.

Reply:
Actually I don't track clicking 'x', but at any rate, because this is a toggle, we don't differentiate opening/closing.  

Rather, my idea was that by focusing on unique events (rather than total), we could only count the first click on a bar for any category, which would always be the first open for that category.

Reply:
Oh ok! I think that's still fine. So we're basically finding out

1) Do people know this is clickable?
2) Do they look at a course and which one?
3) Do they actually click from the course explorer to learn more?
4) Or do they go through the grade bands?

Reply:
Correct.